scripts/check-diff.sh

```diff
@@ -185,7 +185,7 @@ function set_environment_variables {
 	fi
 
 	if [ -z "$ESLINT_CONFIG" ]; then
-		ESLINT_CONFIG="$( upsearch .eslintrc )"
+		ESLINT_CONFIG="$( upsearch .eslintrc* )"
```

Comment:
@adekbadek Were you able to verify that the wildcard selector actually works with `upsearch`?

Reply:
I checked locally with `.eslintrc` and `.eslintrc.js`, and both worked